code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -502,7 +465,7 @@
 		spider.is_busy = TRUE
 		spider.visible_message(span_notice("[spider] begins to lay a cluster of eggs."),span_notice("You begin to lay a cluster of eggs."))
 		spider.stop_automated_movement = TRUE
-		if(do_after(spider, 100, target = get_turf(spider)))
```

Comment:
this isnt 5 seconds like what is said in the pr body this is 10

Reply:
Are you sure?  I thought this timer goes down by 20 a second.